srcpkgs/xmenu/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'xmenu'
+pkgname=xmenu
+version=4.5.5
+revision=1
+build_style=gnu-makefile
+make_build_args="LOCALINC=$XBPS_CROSS_BASE/usr/include \
+	LOCALLIB=$XBPS_CROSS_BASE/usr/lib \
+	FREETYPEINC=$XBPS_CROSS_BASE/usr/include/freetype2 \
+	LDFLAGS+=\$(LIBS) CFLAGS+=\$(INCS)"
```

Comment:
```suggestion
make_build_args="LOCALINC=$XBPS_CROSS_BASE/usr/include
 LOCALLIB=$XBPS_CROSS_BASE/usr/lib
 FREETYPEINC=$XBPS_CROSS_BASE/usr/include/freetype2
 LDFLAGS+=\$(LIBS) CFLAGS+=\$(INCS)"
```

No need to use backslashes. And indentation should be spaces.

Reply:
Got it ~ Removed the `\`'s and replaced tabs with spaces.